bridge/client/app/_views/ktb-integration-view/ktb-integration-view.component.scss

```
@@ -1,11 +1,53 @@
-#integration-container {
-  ::ng-deep p,
-  ul,
-  ol {
-    margin: 0;
-  }
-  h2 {
-    margin-top: 0.3em;
-    margin-bottom: 0.3em;
+@import 'client/variables';
+@import 'node_modules/@dynatrace/barista-components/core/src/style/variables';
```

Could you change it to a relative import?
```suggestion
@import '../../../variables';
@import '~@dynatrace/barista-components/core/src/style/variables';
```